scanomatic/image_analysis/grid_array.py

```diff
@@ -31,15 +31,15 @@ def _analyse_grid_cell(
     analysis_job_model=None
 ):
     save_extra_data = grid_cell.save_extra_data
-
-    grid_cell.source = _get_image_slice(im, grid_cell).astype(np.float64)
-    if grid_cell.source is None:
+    source = _get_image_slice(im, grid_cell)
```

Comment:
why should the `grid_cell` not be updated.

Reply:
fixed so grid_cell still get updated.